src/others/helpers/googleMapsApi.ts

```diff
@@ -0,0 +1,10 @@
+const GOOGLE_API_KEY = 'AIzaSyDa5tT4LZqRjRnbUO5BqHsZNYqG8dIjbqY';
```

Comment:
Is it possible to hide it somehow? Probably not, but it's common case on front, right :)

Reply:
Right, we'll have restrictions by the origin of the call. Then only our dev and prod envs should be able to call it.